packages/remdx/types.tsx

```diff
@@ -1,6 +1,5 @@
 import { MDXProvider } from '@mdx-js/react';
```

Comment:
here you're using a type from `@mdx-js/react` in the types which are exposed in `client.d.ts`, but this is only a `devDependency`:

https://github.com/nkzw-tech/remdx/blob/2849d4bfd21a97cfd5a44b84808fa06e7f95b443/packages/remdx/package.json#L13

Reply:
Ah, this is because the project is shipped with bundled deps except for React. It makes the package lighter and faster to install.

It's not ideal but maybe it makes sense to just copy paste the MDXProvider types into this file and ship it without requiring `@mdx-js/react` to be installed as a dep.

Reply:
Done in d93f92457fe9715b4f6000be292ace95e37b2810